bc/users/models.py

```diff
@@ -9,7 +9,7 @@ class User(AbstractDateTimeModel, AbstractUser):
         help_text="The email address of the user.",
         unique=True,
     )
-    activation_key = models.CharField(max_length=40, default="")
+    activation_key = models.CharField(max_length=64, default="")
```

Comment:
I _think_ if we go with the approach in my earlier comment, we may not need this?

Reply:
You're right!